src/xrUICore/Cursor/UICursor.cpp

```diff
@@ -107,19 +107,11 @@ Fvector2 CUICursor::GetCursorPositionDelta()
 void CUICursor::UpdateCursorPosition(int _dx, int _dy)
 {
     vPrevPos = vPos;
-    if (m_b_use_win_cursor)
-    {
-        Ivector2 pti;
-        IInputReceiver::IR_GetMousePosReal(pti);
-        vPos.x = (float)pti.x * (UI_BASE_WIDTH / (float)Device.m_rcWindowClient.w);
-        vPos.y = (float)pti.y * (UI_BASE_HEIGHT / (float)Device.m_rcWindowClient.h);
-    }
-    else
-    {
-        float sens = 1.0f;
-        vPos.x += _dx * sens;
-        vPos.y += _dy * sens;
-    }
+
+    float sens = 1.0f;
```

Comment:
Does it works normally when the game is running in windowed mode? Are you sure the game cursor and the system cursor won't live with own lives?

Reply:
I tested this code and I didn't see anything strange.